apps/console/src/features/userstores/components/user-stores-list.tsx

```diff
@@ -162,20 +177,24 @@ export const UserStoresList: FunctionComponent<UserStoresListPropsInterface> = (
     const showDeleteConfirm = (): ReactElement => (
         <ConfirmationModal
             onClose={ closeDeleteConfirm }
-            type="warning"
+            type={ type ? type : "warning" }
```

Comment:
Can replace with [nullish-coalescing operator](https://www.typescriptlang.org/play/?#example/nullish-coalescing)
```suggestion
            type={ type ?? warning" }
```